mrp_production_auto_validate/models/mrp_bom.py

```diff
@@ -15,24 +15,9 @@ class MrpBom(models.Model):
             "This behavior is available only if the warehouse is configured "
             "with 2 or 3 steps."
         ),
-        default=False,
-    )
-    mo_auto_validation_warning = fields.Char(
-        string="Order Auto Validation (warning)",
-        compute="_compute_mo_auto_validation_warning",
     )
 
     @api.onchange("type")
     def onchange_type_auto_validation(self):
         if self.type != "normal":
-            self.mo_auto_validation = self.mo_auto_validation_warning = False
-
-    @api.depends("mo_auto_validation")
-    def _compute_mo_auto_validation_warning(self):
-        for bom in self:
-            bom.mo_auto_validation_warning = False
-            if bom.mo_auto_validation:
-                bom.mo_auto_validation_warning = _(
-                    "The Quantity To Produce of an order is now "
-                    "restricted to the BoM Quantity."
-                )
+            self.mo_auto_validation = False
```

Comment:
We could take the opportunity to convert this into a computed field with store=True, readonly=False and precompute=True